src/plugins/intel_npu/src/al/include/intel_npu/config/npuw.hpp

```diff
@@ -65,4 +66,110 @@ DEFINE_OPT(NPUW_DUMP_SUBS, std::string, "", npuw::dump::subgraphs, CompileTime);
 DEFINE_OPT(NPUW_DUMP_SUBS_ON_FAIL, std::string, "", npuw::dump::subgraphs_on_fail, CompileTime);
 DEFINE_OPT(NPUW_DUMP_IO, std::string, "", npuw::dump::inputs_outputs, RunTime);
 DEFINE_OPT(NPUW_DUMP_IO_ITERS, bool, false, npuw::dump::io_iters, RunTime);
+DEFINE_OPT(NPUW_LLM, bool, false, npuw::llm::enabled, CompileTime);
+DEFINE_OPT(NPUW_LLM_MAX_PROMPT_LEN, uint32_t, 1024, npuw::llm::max_prompt_len, CompileTime);
+DEFINE_OPT(NPUW_LLM_MIN_RESPONSE_LEN, uint32_t, 128, npuw::llm::min_response_len, CompileTime);
+
+namespace npuw {
+namespace llm {
+struct ModelDesc {
+    std::string type;
+    std::string name_or_path;
+    int num_key_value_heads;
+};
```

Comment:
Is it passed like 
`"NPUW_LLM_MODEL_DESC" : {"type", "name_or_path", num_key}`?
If so, I don't see a big issue with it. Otherwise it could be 3 separate properties I guess

Reply:
Just answered a comment here. If it's OK, can be removed

Reply:
I'd love to see an example on this too, I guess we'll see it in the associated GenAI PR.

Reply:
"NPUW_LLM_MODEL_DESC" : "type:llama,name_or_path:./llama.xml,num_key_value_heads:32"